src/utilities.c

```diff
@@ -0,0 +1,28 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   utilities.c                                        :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: u413q <u413q@student.42.fr>                +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/10/29 11:48:18 by u413q             #+#    #+#             */
+/*   Updated: 2023/10/29 17:56:09 by u413q            ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "utilities.h"
+
+float	ft_degree_to_radian(float degrees)
+{
+	return (degrees * M_PI / 180.0);
```

Comment:
Performance improvement: define a conversion constant to multiply with.

E.G: # define RAD 0.01745329251

Reply:
added